src/tests/multihost/ad/conftest.py

```diff
@@ -29,6 +29,18 @@ def pytest_configure():
     pytest.num_clients = 1
     pytest.num_others = 0
 
+# ######## Helper functions ####################
+
+
+def _adleave(client):
+    """ Disjoin AD
+    @param client: sssdTools instance of the client
+    """
+    client.disjoin_ad()
+    client.multihost.run_command('kdestroy -A')
```

Comment:
Let's change this so it checks to see if the file is gone, realm leave should remove the keytab, and we should know if *that* behavior has changed.

Reply:
So we should check that the file /etc/krb5.keytab does not exist instead of deleting it?

Reply:
Done.